### ISSUE TITLE
The referenced component 'SharpDX.D3DCompiler' could not be found

### ISSUE THREAD

**Issue by gittestdim** [Microsoft/RoomAliveToolkit#53]
CalibrateEnsemble

Visual Studio 2017 
updated:
MathNet.Nu;erics v3.17.0
ScharpDX v3.1.1

The referenced component 'SharpDX.D3DCompiler' could not be found ...

The referenced component 'SharpDX.DXGI' could not be found.	ProjectorServer			
The referenced component 'SharpDX.Direct2D1' could not be found.	CalibrateEnsemble			
The referenced component 'SharpDX.Direct3D11' could not be found.	CalibrateEnsemble			
The referenced component 'SharpDX.DXGI' could not be found.	CalibrateEnsemble			
The referenced component 'SharpDX.Direct2D1' could not be found.	ProjectorServer			

Properties path = empty

How can we update those references ? Can someone help om this issue?

**Comment by thundercarrot** [comment]
It appears that these other SharpDX libraries have been moved to their own separate NuGet listings. Go to "Manage NuGet Packages for Solution..." and search for "SharpDX", and you will find SharpDX.D3DCompiler, SharpDX.DXGI, etc.

Also note that in updating to SharpDX 3, SharpDX.Matrix is now SharpDX.Mathematics.Matrix; same for Vector3. So you will many compilation errors in updating to SharpDX 3.1.1.

I will make these changes soon. In the meantime, consider rolling back to the older SharpDX if you can.

**Comment by gittestdim** [comment]
Solved!
The problem: in documentation I found that building project RoomAliveKinectServer would automaticly download SharpDX 2.6.3, which it did, (RoomAliveKinectServer ran fine), but gave nothing but reference problems in the ProCamEnsembleCalibration solution. (As you pointed out, upgrading SharpDX did not help).
Recloning roomalive and using NuGet to reference SharpDX 2.6.3 finally did the trick.
Thanks!
